tests/small_hello_world.c

```diff
@@ -0,0 +1,6 @@
+#include <emscripten/html5.h>
+
+int main()
+{
+	emscripten_console_log("hello!");
+}
```

Comment:
For testing, I'd like at minimum to see a new test in `core` that shows minimal mode works in all optimization levels, and in `browser` we should see that it runs in pthreads mode too.

Reply:
Pthreads mode will come in later. Benchmarks are also uninteresting at this point for me, there is nothing really that would be any slower than the regular runtime, and there is currently no definite reason that this would be any faster either.

Reply:
There are now functional tests in core and browser suites.

Reply:
The benchmark suite also tests code size. But we can add that later too.